installer/scripts/genRubyInstaller.sh

```diff
@@ -67,24 +67,26 @@ echo "%Files" > $OUTPUT_FILE
 # For each destination file, generate the appropriate lines for InstallBuilder
 
 NEW_BASE_DIR=""
+saveIFS="$IFS"; IFS=$'\n'
```

Comment:
Reasons for the changes in this file?

Reply:
the "find" CMD with look recursively for all files/dir, if a filename contains a whitespace it will ignore the rest of the filename.
ex: ./test/my test.txt
will be : ./test/my
so ./test/my was considered as directory, this issue is very known in bash scripting and its very nasty to catch.